### ISSUE TITLE
Code example for post in TP is unlikely to work.

### ISSUE THREAD

**Issue by davidkel** [hyperledger/composer#3188]
On the page
https://hyperledger.github.io/composer//integrating/call-out
there is an example which shows getting a transaction, then just posting that transaction. However this example is likely to fail with an error similar to
```
Model violation in instance org.acme.test.DoTxn#c4208296-6cb6-47bb-8bef-d0a2044cc1a5 class org.acme.test.ID has value Resource {id=org.acme.ID#U1} expected a Relationship.
```
Which is not helpful, but is an error message coming from the serializer as it disallows resources for relationships and most transaction objects are going to have dereferenced relationships. The call to post is like this
```
return post( url, postTransaction)
```
Our system tests do this
```
return post( url, postTransaction, {permitResourcesForRelationships: true, deduplicateResources: true})
```
which would solve the problem, so it would seem to me there are 2 document issues here
- what the returned error message means to a user is not clear or maybe not easy to find and the available serializer options to address this
- the example for post is technically correct but likely to fail if people copy the post call.
- This applies to the 0.16.x branch and master.

**Comment by nikkkola** [comment]
@davidkel Do you mean that the sample in docs needs to be changed to be the same as the system tests, i.e. to add the options to the post call or to add a note underneath saying the example would fail?

**Comment by davidkel** [comment]
This is also applicable to `next` until such time as request is made available, however we should change it in next as well just in case request doesn't get made available for the 1.0 release
https://hyperledger.github.io/composer/next/integrating/call-out

**Comment by EdProsser** [comment]
Fixed as of update to the doc.
